lib/cli.js

```diff
@@ -18,11 +20,34 @@ async function main(cli, { policyDetails, setupProjectFn }) {
   } else {
     const currentDate = processDate(cli.flags.currentDate) || undefined;
 
+    let policies = {
+      primary: PRIMARY_POLICY,
+    };
+    if (cli.flags.configFile) {
+      let configFile = fs.readFileSync(cli.flags.configFile, 'utf-8');
+      let customPolicies = JSON.parse(configFile);
```

Comment:
because "custom policies" is a specific term, I would rename this variable "configuredPolicies"